Access API/AccessAPI/DTOs/VAResultDTO.cs

```diff
@@ -1,12 +1,7 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-
-namespace Access_API
+﻿namespace Access_API
 {
     public class VAResultDTO
     {
-        public string response; //TODO update this with whatever the response is
+        public string Response; // TODO update this with whatever the response is
```

Comment:
Check warning siger "Non-nullable field 'Response' must contain a non-null value when exiting constructor. Consider declaring the field as nullable."